src/modules/application/components/providers/providers.test.tsx

```diff
@@ -66,4 +72,9 @@ describe('<Providers /> component', () => {
         expect(screen.getByTestId('dialog-provider-mock')).toBeInTheDocument();
         expect(screen.getByTestId('dialog-root-mock')).toBeInTheDocument();
     });
+
+    it('wraps children with the FeatureFlagsProvider', () => {
```

Comment:
petty but just to stay consistent:

```suggestion
    it('renders the FeatureFlagsProvider with children', () => {
```

Reply:
fixed